signer/grpc_server.go

```diff
@@ -94,8 +94,8 @@ func (rpc *GRPCServer) TransferLeadership(
 }
 
 func (rpc *GRPCServer) GetLeader(
-	ctx context.Context,
-	req *proto.CosignerGRPCGetLeaderRequest,
+	_ context.Context,
+	_ *proto.CosignerGRPCGetLeaderRequest,
```

Comment:
When every parameter is `_`, you can omit the names altogether: `GetLeader(context.Context, *proto.CosignerGRPCGetLeaderRequest) (...) { }`.